jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-2a.yaml

```diff
@@ -55,45 +80,50 @@
           description: Mandatory, version of the cloud to be installed as integer
 
       - string:
-          name: scenario_url
-          default: https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/scenarios/cloud$cloud_version/qa/$ssl_type/qa-scenario-2a.yaml
-          description: Location of scenario yaml file
+          name: scenario_file
+          default: qa-scenario-2a.yaml
+          description: Scenario YAML file name
 
       # Parameters for qa_crowbarsetup.sh
       - string:
           name: cloudsource
           default: develcloud$cloud_version
+
       - string:
           name: TESTHEAD
           default: "1"
           description: if non-empty, test latest version from Devel:Cloud:Staging
+
       - string:
           name: hacloud
           default: "1"
           description: By default we do not want HA configured and installed
+
       - string:
           name: clusterconfig
           default: services=2,data=2,network=2
           description: HA configuration for clusters. Make sense only if hacloud=1
+
       - string:
           name: nodenumber
           default: "7"
           description: Number of nodes to use; is scenario specific
+
       - string:
           name: want_ipmi
           default: "true"
-      - string:
-          name: runner_url
-          default: https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/qa_crowbarsetup.sh
-          description: The runner must be qa_crowbarsetup.sh file
+          description: Boolean. If 'true' nodes will be rebooted via IPMI
+
       - string:
           name: commands
           default: prepareinstallcrowbar bootstrapcrowbar installcrowbar allocate setup_aliases waitcloud
```

Comment:
I think  order of commands should be setup_aliases waitcloud ->  waitcloud setup_aliases